CMSIS/NN/Source/BasicMathFunctions/arm_elementwise_add_s8.c

```diff
@@ -209,7 +206,7 @@ arm_status arm_elementwise_add_s8(const int8_t *input_1_vect,
         sum = MIN(sum, out_activation_max);
         r4 = (q7_t)sum;
 
-        write_q7x4_ia(&output, __PACKq7(r1, r2, r3, r4));
+        arm_nn_write_q7x4_ia(&output, PACK_Q7x4_32x1(r1, r2, r3, r4));
```

Comment:
Another common feature NN and DSP both need. DSP has __PACKq7
Another candidate for moving it somewhere else ?

Reply:
Yes.. this too along with the read/write ones..